src/MapMaker.js

```diff
@@ -0,0 +1,60 @@
+const { MAP } = require("./view/stringsUI");
+
+class MapMaker {
+  #upperBridge;
+
+  #lowerBridge;
+
+  constructor(playerArr) {
+    this.#upperBridge = [];
+    this.#lowerBridge = [];
+    this.createMap(playerArr);
+  }
+
+  createMap(playerArr) {
+    playerArr.forEach((playerAction, index) => {
+      this.divideBridge(index);
+      this.handlePlayerAction(playerAction);
+    });
+    this.wrapBridge();
+    return this.getBridge();
+  }
+
+  handlePlayerAction({ selectedMove, isCrossBridge }) {
+    if (selectedMove === MAP.UPPER) {
+      this.#upperBridge.push(MAP[isCrossBridge]);
+      this.#lowerBridge.push(MAP.NONE);
+    } else {
+      this.#lowerBridge.push(MAP[isCrossBridge]);
+      this.#upperBridge.push(MAP.NONE);
+    }
+  }
```

Comment:
else를 안쓴다면 이렇게 작성할 수 있을 것 같아요!
```suggestion
  handlePlayerAction({ selectedMove, isCrossBridge }) {
    if (selectedMove === MAP.UPPER) {
      this.#upperBridge.push(MAP[isCrossBridge]);
      this.#lowerBridge.push(MAP.NONE);
      return;
    }
    
    this.#lowerBridge.push(MAP[isCrossBridge]);
    this.#upperBridge.push(MAP.NONE);
  }
```